apps/mull-ui/src/app/pages/profile/other-user-profile/other-user-profile.tsx

```diff
@@ -0,0 +1,59 @@
+import { ISerializedEvent } from '@mull/types';
+import { History } from 'history';
+import React, { useContext } from 'react';
+import { useOtherUserProfileQuery, User } from '../../../../generated/graphql';
+import { sortEventsByDate } from '../../../../utilities';
+import { EventCard, MullBackButton } from '../../../components';
+import ProfileHeader from '../../../components/profile-header/profile-header';
+import UserContext from '../../../context/user.context';
+import './other-user-profile.scss';
+
+export interface OtherUserProfilePageProps {
+  history: History;
+  prevPage: string;
+}
+
+export const OtherUserProfilePage = ({ history, prevPage }: OtherUserProfilePageProps) => {
+  const { data: otherUserProfileData, loading: userProfileLoading } = useOtherUserProfileQuery({
+    variables: {
+      // TODO in TASK-83: get id of other user
+      id: 4,
```

Comment:
⚠I understand you might want to address this in TASK-83, but I feel like it'd be good to do this properly now instead of reworking it later.

Instead of changing the number here in the code. You should set `OtherUserProfilePage` to show up on a parameterized route like `/profile/:id`. And simply fetch the id from the route for your query.

This would be much simpler if the profile page was one component 👀 , but since we're not doing that, we could instead have a wrapper component that shows up when we go to `/profile/:id`. It would then check if :id is the same as the current user id. if it is, then it would show the user profile, otherwise, it would show the otheruserprofile.

If you really think this shouldn't be addressed now, I'm fine with dealing with this later, but I think this will make things simpler in the end.

Reply:
Ah I understand your concern, and you are right - it should be implemented before #284 as it will make the E2E easier to create (and less refactoring later on) as well. However, I do want to implement your suggestion in #270 if you don't mind, as there isn't much to do in that ticket anyway other than the E2E currently.

Reply:
sure, that's fine, I'll assign myself to that ticket to make sure that gets addressed then